libs/libdsp/Kconfig

```diff
@@ -30,4 +30,7 @@ config LIBDSP_PRECISION
 		1 - a little better precision than above, but slowest
```

Comment:
@raiden00pl - I know is is from the original, but can you add more context here? (above what????)

Reply:
I updated this comment. Is it ok ?